pulp_smash/cli.py

```diff
@@ -232,6 +233,11 @@ def machine(self):
 
             if transport == "local":
                 self._machine = plumbum.machines.local
+            elif transport == "kubectl":
+                self._machine = plumbum.machines.local
+                chain = self._machine["kubectl"]["get", "pods"] | \
+                        self._machine["grep"]["-E", "pulp-api-(\w+)-(\w+)"]
+                self._podname = chain().split(" ")[0]
```

Comment:
Are you trying to get only the name of the pod? If so, just add "-o" to grep.

Reply:
yep, i added teh -o. thank you!